src/screens/TreeScreen.tsx

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
```

Comment:
Delete this file for now, not part of task scope and made redundant by `SearchScreen`.
(Make sure to remove it from navigation as well).